source/vibe/data/json.d

```diff
@@ -181,6 +253,20 @@ struct Json {
 	/// ditto
 	long opAssign(long v) { m_type = Type.int_; m_int = v; return v; }
 	/// ditto
+	version (VibeUseJsonBigInt) {
+		BigInt opAssign(BigInt v)
+		{
+			if (v >= long.min && v <= long.max) {
```

Comment:
I think making this choice only in the parser is a better idea. It could be surprising to have code like this trigger an error: `void test(BigInt num) { Json j = num; assert(j.type == Json.Type.bigInt); }`